readthedocs/projects/forms.py

```diff
@@ -285,6 +285,18 @@ def get_all_active_versions(self):
             return all_versions
         return None
 
+    def save(self, commit=True):
```

Comment:
can we use the `pre_save` signal to handle this?

Reply:
Yes of course, just did it the same way like in `ProjectTriggerBuildMixin`

Reply:
Using signals will allow us to not modify upsteam code and keep our changes in `docsitalia`
Let's port the code to signals (maybe we need to figure out how to trigger the build in pre_save but fetch the updated data, IDK if `post_save` will do)